includes/admin/settings/class-settings-bootstrap.php

```diff
@@ -915,6 +915,23 @@ private static function settings_appearance() {
 
 		WPBDP_Admin_Education::add_tip_in_settings( 'table', 'themes' );
 
+        if ( ! is_plugin_active( 'font-awesome/index.php' ) || ( ! class_exists( '\FortAwesome\FontAwesome_Loader' ) ) ) {
+			wpbdp_register_setting(
+				array(
+					'id'    => 'fontawesome-enabled',
+					'name'  => __( 'Font Awesome', 'business-directory-plugin' ),
```

Comment:
Let's get rid of this name line for education. It makes it looks like there will be a setting when there isn't.